skopt/acquisition.py

```diff
@@ -264,3 +267,119 @@ def gaussian_ei(X, model, y_opt=0.0, xi=0.01, return_grad=False):
         return values, grad
 
     return values
+
+
+def gaussian_entropy_search(X, model, space, n_rep_points=50,
```

Comment:
We use a Gaussian Process posterior to approximate `p_min` at each of the representer points, i.e this implementation works only if the posterior is multi-variate gaussian. Sec 2.3 in the entropy search paper (http://jmlr.csail.mit.edu/papers/volume13/hennig12a/hennig12a.pdf)